packages/core/src/dialects/abstract/query-interface-typescript.ts

```diff
@@ -170,6 +170,16 @@ export class AbstractQueryInterfaceTypeScript {
     return schemaNames.map(schemaName => schemaName.schema);
   }
 
+  /**
+   * Show all defined schemas
+   *
+   * @deprecated Use {@link listSchemas} instead.
+   * @param _options
+   */
+  async showAllSchemas(_options?: QueryRawOptions): Promise<string[]> {
+    throw new Error(`showAllSchemas has been deprecated. Use listSchemas instead.`);
```

Comment:
I was thinking more like how we deal with the [deprecation of Model.scope](https://github.com/sequelize/sequelize/blob/5a50f9b8fc0425409f35aad90e74b5590c5024d1/packages/core/src/model.js#L982-L987) where we link to a deprecation in [utils/deprecation](https://github.com/sequelize/sequelize/blob/main/packages/core/src/utils/deprecations.ts) and then call the new function ourselves. Could you update it to that?

Reply:
Updated now.